hawkular-metrics/standalone.xml

```diff
@@ -217,8 +217,13 @@
                     <transaction mode="BATCH"/>
                 </replicated-cache>
             </cache-container>
-            <cache-container name="web" default-cache="dist" module="org.wildfly.clustering.web.infinispan">
+            <cache-container name="web" default-cache="repl" module="org.wildfly.clustering.web.infinispan">
                 <transport lock-timeout="60000"/>
+                <replicated-cache name="repl" mode="ASYNC">
```

Comment:
It follows the recommendations by the cloud enablement team for running JBoss on OpenShift.